src/odemis/driver/technolution.py

```diff
@@ -0,0 +1,965 @@
+        self.scanOffset = model.TupleContinuous((0.0, 0.0), range=((-10.0, -10.0), (10.0, 10.0)), unit='V')
+        self.scanGain = model.TupleContinuous((0.0, 0.0), range=((-10.0, -10.0), (10.0, 10.0)), unit='V')
+
+
+class MPPC(model.Detector):
+    """
+    Represents the camera (mppc sensor) for acquiring the image data.
+    """
+
+    def __init__(self, name, role, parent, **kwargs):
+        """
+        Initializes the camera (mppc sensor) for acquiring the image data.
+
+        :param name:
+        :param role:
+        :param parent:
+        :param kwargs:
+        """
+        super(MPPC, self).__init__(name, role, parent=parent, **kwargs)
+
+        self._server_URL = self.parent._server_url
+
+        self._shape = (8, 8)
+        self.filename = model.StringVA(time.strftime("default--%Y-%m-%d-%H-%M-%S"), setter=self._setFilename)
+        self.dataContent = model.StringEnumerated('empty', DATA_CONTENT_TO_ASM.keys())
+        # NOTE: Do not write real username/password here since this is published on github in plaintext!
+        self.externalStorageURL = model.VigilantAttribute(urlparse('ftp://username:password@example.com/Pictures'),
+                                                          setter=self._setURL)
+        self.acqDelay = model.FloatContinuous(2.0, range=(0, 100000), unit='s', setter=self._setAcqDelay)
+
+        # Cell acquisition parameters
+        self.cellTranslation = model.ListVA([[[50, 50]] * self._shape[0]] * self._shape[1],
+                                            setter=self._setCellTranslation)
+        self.cellDarkOffset = model.ListVA([[0] * self._shape[0]] * self._shape[1], setter=self._setcellDarkOffset)
+        self.cellDigitalGain = model.ListVA([[1.2] * self._shape[0]] * self._shape[1], setter=self._setcellDigitalGain)
+        self.cellCompleteResolution = model.ResolutionVA((800, 800), ((10, 10), (1000, 1000)))
+
+        # TODO K.K. pass right metadata from new simulator
+
+        # Setup hw and sw version
+        # TODO make call set_system_sw_name to new simulator (if implemented)
+        self._swVersion = self._swVersion + ", " + "PUT NEW SIMULATOR DATA HERE"
+        self._hwVersion = self._hwVersion + ", " + "PUT NEW SIMULATOR DATA HERE"
+
+        # Gather metadata from all related HW components and own _meta_data
+        self.md_devices = [self._metadata, self.parent._mirror_descanner._metadata,
+                           self.parent._ebeam_scanner._metadata]
+        self._metadata[model.MD_HW_NAME] = "MPPC" + "/" + name
+        self._metadata[model.MD_SW_VERSION] = self._swVersion
+        self._metadata[model.MD_HW_VERSION] = self._hwVersion
+
+        # Initialize acquisition processes
+        self.acq_queue = queue.Queue()  # acquisition queue with commands of actions that need to be executed.
+        self._acq_thread = threading.Thread(target=self._acquire, name="acquisition thread")
+        self._acq_thread.deamon = False
```

Comment:
Sorry I changed the meaning of deamon and non deamon. I actually thought deamons couldn't die (because of a lack of a soul) but it's the other way around. Because deamon comes from Maxwell's demon working in the background.
I fixed it.